sql/service/src/main/java/org/apache/spark/sql/service/cli/CLIService.java

```diff
@@ -172,65 +172,70 @@ public synchronized void stop() {
   @Deprecated
```

Comment:
Could remove these lines?
![image](https://user-images.githubusercontent.com/5399861/69790420-6e1cca00-11fd-11ea-88d1-cd3b2edbebb1.png)

https://github.com/spark-thriftserver/spark-thriftserver/pull/5/files#diff-3f4a32658d0ebd805386f17786a44547L149-L161

Please see http://issues.apache.org/jira/browse/HIVE-10976 for more details.